addons/survey/static/src/js/survey.js

```diff
@@ -129,6 +129,15 @@ if(!the_form.length) {
         }
     }
 
+    $('.breadcrumb-item a').each(function(key, value) {
+        var valObj = $(value)
+        valObj.click(function(event) {
+            event.preventDefault()
+            $('<input>').attr({type: 'hidden', name: 'breadcrum_redirect', value: valObj.attr('href')}).appendTo('form');
```

Comment:
here the `appendTo` should be to `.js_surveyform` (it should not be an issue but is ugly to add things in form where it is not needed)